### ISSUE TITLE
Obtaining paths.xml

### ISSUE THREAD

**Issue by TheManjia** [bradenhurl/DeepGTAV-PreSIL#22]
Hello! 
Is it possilble for you to host the paths.xml in some file host provider different from the access protected google drive it resides now? 
I've tried to download it and asked permission from the prompt but as of today (a week after) I wasn't able to download it yet

Thank You!

**Comment by ll594282475** [comment]
So do I. Have you got this file?

**Comment by TheManjia** [comment]
Yes, I eventually managed to extract it from GTAV game files by using OpenIV, it is not too much complicated but downloading it with one click is surely more convenient to you, so here it is:

https://drive.google.com/file/d/1_flOtMc4vrdIAmUbW-AWxdItRLqfifJ3/view?usp=sharing

**Comment by TheManjia** [comment]
Reopening as I think @bradenhurl should be the one providing a permalink to this file, I cannot guarantee to keep it forever on my personal Google Drive

**Comment by ll594282475** [comment]
Ok, i have download your version, but&nbsp; I haven't fully replicated the project.&nbsp;
thx a lot again！



---Original---
From: ***@***.***&gt;
Date: Wed, Apr 27, 2022 21:56 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [bradenhurl/DeepGTAV-PreSIL] Obtaining paths.xml (Issue #22)




 
Reopening as I think @bradenhurl should be the one providing a permalink to this file, I cannot guarantee to keep it forever on my personal Google Drive

**Comment by ll594282475** [comment]
thx a lot!!!




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年4月27日(星期三) 晚上6:34
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [bradenhurl/DeepGTAV-PreSIL] Obtaining paths.xml (Issue #22)





 
Yes, I eventually managed to extract it from GTAV game files by using OpenIV, it is not too much complicated but downloading it with one click is surely more convenient to you, so here it is:
 
https://drive.google.com/file/d/1_flOtMc4vrdIAmUbW-AWxdItRLqfifJ3/view?usp=sharing
